name.abuchen.portfolio.tests/src/name/abuchen/portfolio/model/InvestmentPlanTest.java

```diff
@@ -47,6 +47,9 @@ public void setup()
     @Test
     public void testGenerationOfBuyTransaction() throws IOException
     {
+        investmentPlan.setType(InvestmentPlan.Type.BUY_OR_DELIVERY); // explicit
```

Comment:
I removed the comment "explicit now". I don't think it helps the explain the current code (and the change can be seen in the history).